test/token/ERC20/ERC20.behavior.js

```diff
@@ -1,5 +1,7 @@
-const { BN, constants, expectEvent } = require('@openzeppelin/test-helpers');
+const { ethers } = require('hardhat');
 const { expect } = require('chai');
+
+const { BN, constants, expectEvent } = require('@openzeppelin/test-helpers');
```

Comment:
Test migration should remove this from the dependency.

Reply:
I did it and removed unused libs